instances.go

```diff
@@ -39,8 +48,8 @@ type InstancePolicy struct {
 	CreatedAt  *time.Time `json:"creationDate,omitempty"`
 	UpdatedAt  *time.Time `json:"lastUpdated,omitempty"`
 	UpdatedBy  string     `json:"updatedBy,omitempty"`
-	PolicyType string     `json:"policy_type,omitempty"`
-	PolicyData PolicyData `json:"policy_data,omitempty" mapstructure:"policyData"`
+	PolicyType string     `json:"policyType,omitempty"`
+	PolicyData PolicyData `json:"policyData,omitempty" mapstructure:"policyData"`
```

Comment:
Can't change the name of these since they are determined by the server response. These will be empty/nil now whenever you attempt to deserialize a response into thsi struct since the attributes will never exist.